### ISSUE TITLE
Inline chat `Discard` button should have clearer, more comprehensive label

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#200081]
The fable study showed that magnifier users and switch users were surprised by the behavior of the `Discard` button in inline chat. 

Switch users did not expect their input to be cleared and it can be very time consuming for them to retype their request. 

Magnification users were disoriented when the entire widget closed and their focus was moved back to the editor. 

Can we improve the label to indicate what will happen? 

Should we have separate buttons for clearing the input and closing the widget?

cc @isidorn

**Comment by meganrogge** [comment]
What about `Reset` instead of `Discard`?

**Comment by jrieken** [comment]
> Should we have separate buttons for clearing the input and closing the widget?

We have that now. There is a dedicated close gesture in the upper right corner but discarding the last AI change still closes the widget. Tho, I think that's reasonable, you have undo all AI changes and nothing is left

**Comment by meganrogge** [comment]
I think we can close this, tracking the issue where users cannot go between the widget and the editor here https://github.com/microsoft/vscode/issues/202250
